balpy/balpy.py

```diff
@@ -98,7 +98,7 @@ class balpy(object):
 						"ropsten":	{"id":3,		"blockExplorerUrl":"ropsten.etherscan.io"													},
 						"rinkeby":	{"id":4,		"blockExplorerUrl":"rinkeby.etherscan.io"													},
 						"goerli":	{"id":5,		"blockExplorerUrl":"goerli.etherscan.io"													},
-						"optimism":	{"id":10,		"blockExplorerUrl":"optimistic.etherscan.io"													},
+						"optimism":	{"id":10,		"blockExplorerUrl":"optimistic.etherscan.io"												},
```

Comment:
Is this just `diff` being weird, or did something actually change here?

Reply:
I think this is just the diff being weird 😬